### DISCUSSION TITLE
Mode auto et contenu chargé en ajax

### DISCUSSION THREAD

**Discussion by tromeurf** [AmauriC/tarteaucitron.js#898] [Q&A]
Bonjour,

J'ai un contenu qui comporte une iframe youtube et qui est chargé en ajax dans une modale.
Le mode automatique fonctionne bien sur mon site mais je n'ai pas trouvé le moyen de rappeler ce mode auto pour bloquer la vidéo dans la modale.
Y a t-il un moyen simple ? une fonction à appeler dans mon callback ajax ?

Merci pour votre aide

**Comment by AmauriC** [comment]
Le mode auto n'est pas (encore?) compatible avec le chargement dynamique : /
